.rubocop.yml

```diff
@@ -41,6 +46,14 @@ Naming/FileName:
 
 # --- Style ---
 
+# Pointless change
+Style/FormatStringToken:
+  Enabled: false
+  
+# Had to turn this off for many files
```

Comment:
again - "why?".    "Loops were easier to understand without favoring next statements" ?